charts/datadog/templates/_otel_agent_config.yaml

```diff
@@ -35,13 +35,9 @@ otel-config.yaml: {{- if .Values.datadog.otelCollector.config }} {{ toYaml .Valu
     service:
       pipelines:
         traces:
-          receivers: [otlp]
-          processors: [batch]
-          exporters: [datadog/connector]
-        traces/otlp:
           receivers: [otlp]
           processors: [infraattributes, batch]
-          exporters: [datadog]
+          exporters: [datadog, datadog/connector]
```

Comment:
do this change require a minimum agent version?

Reply:
This change doesn't require a minimum version.